addons/loyalty_delivery/tests/test_loyalty_delivery.py

```diff
@@ -0,0 +1,99 @@
+
+from odoo.tests import common, Form
+from odoo.exceptions import ValidationError
+
+@common.tagged('post_install', '-at_install')
+class TestLoyaltyDeliveryCost(common.TransactionCase):
+
+    def setUp(self):
+        super(TestLoyaltyDeliveryCost, self).setUp()
```

Comment:
Avoid using setUp, use `setUpClass(cls)` instead.
It's a good habit for tests, as it avoids doing the same setup multiple times (when there are multiple test methods).